src/components/nmrfile-browser/NMRFileBrowser.tsx

```diff
@@ -0,0 +1,507 @@
+                      alignSelf: 'center',
+                      textOverflow: 'ellipsis',
+                      whiteSpace: opened === entry.groupName ? '' : 'nowrap',
+                      wordWrap: 'break-word',
+                      width: '1em',
+                      flex: 1,
+                    }}
+                  >
+                    {entry.groupName}
+                  </span>
+                  {TableRef.current?.offsetWidth === undefined ||
+                    (TableRef.current?.offsetWidth > 300 && (
+                      <div
+                        style={{
+                          alignSelf: 'center',
+                        }}
+                      >
+                        {formatDistanceToNowStrict(entry.lastModified, {
+                          addSuffix: true,
+                        })}
+                      </div>
+                    ))}
+                  <div
+                    style={{
+                      display: 'flex',
+                      alignSelf: 'center',
+                      gap: '2px',
+                    }}
+                  >
+                    {getLink(
+                      entry,
+                      TableRef.current?.offsetWidth === undefined ||
+                        TableRef.current?.offsetWidth > 500,
+                    )}
+                  </div>
+                  {TableRef.current?.offsetWidth === undefined ||
+                    (TableRef.current?.offsetWidth > 500 && (
+                      <div
+                        style={{
+                          alignSelf: 'center',
+                        }}
+                      >
+                        {getDownloadLink(entry)}
+                      </div>
+                    ))}
+                </div>
+              </div>
+            </Collapsible.Trigger>
+            <Collapsible.Content css={style.content}>
+              <Table
+                striped
+                css={css({
+                  width: '100%',
+                })}
+                compact
+              >
+                {entry.children.map((child: NMREntryChild) => (
+                  <Table.Row
+                    key={child.id}
+                    style={{
+                      height: '10px',
+                      padding: '0 !imporant',
+                    }}
+                  >
+                    <ValueRenderers.Text
+                      value={formatISO9075(child.lastModified)}
+                    />
+                    <ValueRenderers.Text value={getDim(child)} />
+                    <ValueRenderers.Text value={getType(child)} />
+                    <ValueRenderers.Component>
+                      <div
+                        dangerouslySetInnerHTML={{
+                          __html: child.solvent.replaceAll(
+                            /(\d+)/g,
+                            '<sub>$1</sub>',
+                          ),
+                        }}
+                      />
```

Comment:
@lpatiny Can we use react-mf here?

Reply:
Unfortunately we can not because solvent can be typically DMSO-d6 and this is not a MF  I think just number in subscript fulfil our needs in this case.

![image](https://github.com/user-attachments/assets/d49936ce-1a56-4803-b2fd-e2fc4dc62e83)

Reply:
Well we should still not use `dangerouslySetInnerHTML`, but create a specific component